src/test/java/api/support/builders/ClaimItemReturnedRequestBuilder.java

```diff
@@ -1,7 +1,7 @@
 package api.support.builders;
 
-import static org.folio.circulation.resources.ClaimItemReturnedResource.COMMENT;
-import static org.folio.circulation.resources.ClaimItemReturnedResource.ITEM_CLAIMED_RETURNED_DATE;
+import static org.folio.circulation.domain.representations.ChangeItemStatusRequest.COMMENT;
+import static org.folio.circulation.domain.ClaimItemReturnedRequest.ITEM_CLAIMED_RETURNED_DATE;
```

Comment:
Similarly, I don't think that API test builders should rely on the internal domain implementation